uint8 NUM_ACTUATORS_DIRECT = 16
uint64 timestamp			# timestamp in us since system boot
uint32 nvalues				# number of valid values
float32[16] values			# actuator values, from -1 to 1
